### ISSUE TITLE
bug: configuration file excludes are no longer honored in 1.9.0

### ISSUE THREAD

**Issue by nwsparks** [aquasecurity/tfsec#1597]
**Describe the bug**

if the config file is present in .tfsec/config.yml it is not honored when running tfsec

likewise it is no longer honored when manually defining it such as `tfsec --config-file tfsec.yml`

this is only an issue in 1.9.0

reverting to 1.8.0 it is correctly working again.

**Comment by liamg** [comment]
I'll look at this now, though GitHub is till having a partial outage which has broken actions so it mayb not be fixed until tomorrow a.m. :(

**Comment by owenrumney** [comment]
Hey @nwsparks  can you provide the exclude section of your config? Or a sample of it. 

Interested specifically in what IDs you using

**Comment by owenrumney** [comment]
we've found the issue - new release will be coming shortly when the build completes. Apologies @nwsparks

**Comment by nwsparks** [comment]
thanks for the fast responses. here is our config file with some comments redacted.

```
---
exclude:
  - aws-s3-specify-public-access-block
  # comment with link http://google.com

  - aws-s3-enable-bucket-logging
  # comments

  - aws-vpc-no-public-egress-sg
  - aws-vpc-no-public-egress-sgr
  # comments
  # comments

  - aws-vpc-add-description-to-security-group
  - aws-vpc-add-description-to-security-group-rule
  # comment

  - aws-elb-http-not-used
  # comment

  - aws-ecr-enforce-immutable-repository
  # cant reassign latest tag if enabled. open github issue: https://github.com/aws/containers-roadmap/issues/878

  - aws-lambda-enable-tracing
  # comment

  - aws-rds-enable-performance-insights
  - aws-rds-enable-performance-insights-encryption
  # comment

  # we use the default KMS key for the below items.
  - aws-cloudwatch-log-group-customer-key
  - aws-documentdb-encryption-customer-key
  - aws-dynamodb-table-customer-key
  - aws-ebs-encryption-customer-key
  - aws-ecr-repository-customer-key
  - aws-redshift-encryption-customer-key
  - aws-ssm-secret-use-customer-key
  - aws-kinesis-enable-in-transit-encryption # misleading. actually a complaint about no cmk
  - aws-cloudwatch-log-group-customer-key
  - aws-ecr-repository-customer-key
  - aws-dynamodb-table-customer-key
  - aws-s3-encryption-customer-key
  - aws-ssm-secret-use-customer-key
  - aws-sqs-enable-queue-encryption # misleading. actually a complaint about no cmk
  - aws-sns-enable-topic-encryption # misleading. actually a complaint about no cmk
  - aws-rds-encrypt-cluster-storage-data # misleading. actually a complaint about no cmk
  # end cmk

  - github-repositories-enable_vulnerability_alerts
  # we use snyk, so these are redundant and not required.

  ## The following rules were introduced with the tfsec upgrade from 0.62.0 to 1.8.0
  ## These are high priority to be reviewed.
  - aws-iam-no-policy-wildcards
  - aws-s3-block-public-acls
  - aws-s3-block-public-policy
  - aws-s3-enable-bucket-logging
  - aws-s3-enable-versioning
  - aws-s3-ignore-public-acls
  - aws-s3-no-public-buckets
  - aws-s3-specify-public-access-block
  ## End v1.8.0 exclusion review
```

**Comment by nwsparks** [comment]
that would be neat. i don't think we would use that ourselves as we just have a single repo which contains all our tf code and this is stored in the root. i could see teams having multiple repos for modules or such wanting this though.

**Comment by liamg** [comment]
Thanks for the report, this should be resolved in `v1.9.1` (building now).
